python/tests/test_scenario_executor_events.py

```
@@ -474,3 +474,134 @@ def _on_event(event: ScenarioEvent) -> None:
+        return "Hey, how can I help you?"
+
+
+class PlainAgent(AgentAdapter):
+    async def call(self, input: AgentInput) -> str:
+        return "Hey, how can I help you?"
+
+
+class BlankNameAgent(AgentAdapter):
+    name = "   "
+
+    async def call(self, input: AgentInput) -> str:
```

Ruff (0.16.2)

[error] Function argument `input` is shadowing a Python builtin

(A002)